### PR TITLE
Using unit tests instead of Property tests

### PR DESCRIPTION
**Pull request by Titas212**

### CODE REVIEW

File: Learning-scala/src/main/scala/JTS/Polygons.scala [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package jts
+package JTS
```
Comment (PKJonas) [inline Learning-scala/src/main/scala/JTS/Polygons.scala:1]: Package names should be lowercase https://docs.scala-lang.org/style/naming-conventions.html#packages

File: Learning-scala/src/main/scala/JTS/Polygons.scala [hunk 2]
```diff
@@ -8,15 +8,15 @@ import scala.annotation.tailrec
...
 
-object LithuaniaPoly {
+object Polygons {
 
   val geometryFactory = new GeometryFactory()
 
   def parsePolyFile(filename: String): Either[String, CoordinateList] = {
-    val file        = new File(filename)
-    val scanner     = new Scanner(file)
+    val file = new File(filename)
```
Comment (PKJonas) [inline Learning-scala/src/main/scala/JTS/Polygons.scala:16]: Consider adding a scalafmt config to the project

File: Learning-scala/src/test/scala/JTS/TestingPolygons.scala [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package JTS
```
Comment (PKJonas) [inline Learning-scala/src/test/scala/JTS/TestingPolygons.scala:1]: Same as above

File: Learning-scala/src/test/scala/JTS/TestingPolygons.scala [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+  describe("pointsCheck") {
+    it("Checking if correct amount of points were generated") {
+      polygons.map(x => assert(Polygons.generatePointsInsidePolygon(x, 100).size == 100))
+    }
+  }
+
+  describe("parseCheck") {
+    it("Checking if parsing was successful") {
+      countries.map(x => Polygons.parsePolyFile(x)).map(x => assert(x.isRight))
```
Comment (PKJonas) [inline Learning-scala/src/test/scala/JTS/TestingPolygons.scala:27]: You already throw if `val polygons` isn't successfully assigned

File: Learning-scala/src/test/scala/JTS/TestingPolygons.scala [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    }
+  }
+
+  describe("validatingPoints") {
+    it("Checking, if points generated are inside polygon") {
+      for
```
Comment (PKJonas) [inline Learning-scala/src/test/scala/JTS/TestingPolygons.scala:33]: `for-comprehension` isn't really needed here?

**Review by PKJonas — APPROVED** [review]

File: Learning-scala/src/test/scala/JTS/TestingPolygons.scala (hunk 5 above)
Comment (Titas212) [reply Learning-scala/src/test/scala/JTS/TestingPolygons.scala:33]: Yes, it's just for better readability

**Pull request merged**